client-lib/go/client/jvs_client.go

```diff
@@ -50,12 +59,50 @@ func NewJVSClient(ctx context.Context, config *JVSConfig) (*JVSClient, error) {
 	}
 
 	return &JVSClient{
-		config: config,
-		keys:   cached,
+		config:           config,
+		keys:             cached,
+		forbidBreakglass: config.ForbidBreakglass,
```

Comment:
You already have `config` as a field, why extracting this as a separate field?

Reply:
Thank you for catching it! We don't need it. At first, I just followed the java code logic and extracted the boolean as a separate filed https://github.com/abcxyz/jvs/blob/main/client-lib/java/src/main/java/com/abcxyz/jvs/JvsClient.java#L44